.github/workflows/ci.yml

```diff
@@ -10,7 +10,7 @@ on:
       - '.gitignore'
 env:
   node_version: "14.x"
-  tf_version: "0.14.7" # must match value in terraform-iac/*/app/main.tf
+  tf_version: "1.0.0" # must match value in terraform-iac/*/app/app-*.tf
```

Comment:
Why the filename change?

Reply:
I changed all the file names so that it's obvious which file you're updating. If you have "main.tf" open, which one are you edting? dev? prd? etc.

Reply:
I won't block on that, but I'm not a fan. Does your editor not give you a nice tree structure?

<img width="182" alt="directory" src="https://user-images.githubusercontent.com/7625337/122591211-dcafbb80-d01f-11eb-9106-0cb613e99700.png">

Reply:
It does, but when you have one file open in the tab is shows `main.tf` unless you have more than one `main.tf` file open then it'll show `dev/main.tf` and `prd/main.tf` and honestly I look at the tabs. I'm using IntelliJ and I assume WebStorm is the same. I have a project structure on the side, but I only use that to open files, not to see which ones I'm currently editing.

Reply:
At first, I was going to make the argument that this ruins scripting like:
```sh
cd app && terraform init -upgrade && cd ../setup && terraform init -upgrade && cd ..
```
but it really doesn't since TF works on directories rather than individual filenames.

So, my main objection is based on personal preference. 🤷

Reply:
Yeah, the directories stay the same (i.e. I'm not changing `/terraform-iac/dev/app` to `/terraorm-iac/app-dev` or whatever). And this is my personal preference as well. I'm happy to change it back to `main.tf`, just know that all my projects will have the individual files named `app-dev.tf` etc. 😅

Reply:
@byu-oit/devops-tooling - Anyone else want to weigh in on the bikeshedding?

Reply:
I'm pretty torn on this one actually. On one hand, conventionally the main.tf file is the "entrypoint" for developers to start understanding the code and since this module is a template for other developers on campus, I'm tempted to keep it the same so that campus developers follow a similar convention. On the other hand, renaming the files may help developers using JetBrains IDEs. 🤷🏻‍♂️ Personally, I'd keep it how it was for consistency with other terraform modules.

Reply:
To your point @stuft2, I would argue that these terraform files are not modules. Yes the internal "/app" one is, but it's not being used externally, only internally. And "app-dev.tf" is a terraform configuration. You can name those whatever you want since terraform looks at the directory you're in and not the names of the files it tries to use.

Our byu-oit terraform modules I'm perfectly fine keeping named "main.tf" since they are external and follow the standard practices etc.

Reply:
For now we're going to punt on this discussion and keep `main.tf` as the file names

Reply:
Looks like in 1d483d5, we're deferring this conversation for a follow-up PR.